pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
> if we set this to privileged, there is no point to the hostmount SCC... recycler pods will fail until the recycler service account is added to the privileged SCC

Exactly.  You will never validate with the privileged SCC since it isn't permissioned to this account OOTB.  

> Yes, this was necessary

What is requiring privileged access and can it be achieved by just setting the `hostmount` user strategy to `RunAsAny` and running this pod with `RunAsUser: 0`? (possibly renaming that SCC to indicate is is specifically an infra scc and used by the recycler?)